src/pages/admin/categories/CategoryList/index.js

```diff
@@ -1,7 +1,154 @@
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>6</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>7</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>8</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>9</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>10</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
+                    <tr>
+                      <td  id={style.tBodyStyle}>11</td>
+                      <td id={style.tBodyStyle}>Data Structures</td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><FaRegEdit size="20px"/></Button></td>
+                      <td id={style.tBodyStyle1}><Button className={style.designButton}><RiDeleteBin2Fill size="30px" color="#db7771"/></Button></td>
+                    </tr>
```

Comment:
instead of displaying one by one, please make a variable containing all dummy data and loop through it. in the loop, display each of the items